src/main/scala/scorex/core/validation/ModifierValidator.scala

```diff
@@ -94,11 +94,12 @@ object ModifierValidator extends ScorexLogging  {
   val success: Valid[Unit] = Valid(())
 
   private def msg(descr: String, e: Throwable): String = msg(descr, Option(e.getMessage).getOrElse(e.toString))
+
   private def msg(description: String, detail: String): String = s"$description: $detail"
 }
 
 /** This is the place where all the validation DSL lives */
-case class ValidationState[T](result: ValidationResult[T], strategy: ValidationStrategy)(implicit e: BytesEncoder) {
+case class ValidationState[T](result: ValidationResult[T], strategy: ValidationStrategy)(implicit e: ScorexEncoder) {
```

Comment:
i'd like to drop `ScorexEncoder` here and use just `id.toString` method as `id` is not an array of bytes anymore